build.gradle

```diff
@@ -346,7 +346,6 @@ task integrationTest(type: Test) {
 check.dependsOn integrationTest
 
 dependencies {
-    implementation 'jakarta.annotation:jakarta.annotation-api:1.3.5'
```

Comment:
That's is right, it comes from the recent protobuf-related changes when the `@Generated` annotation was introduced, thanks @Xtansia